examples/nlp_seq/README.md

```diff
@@ -0,0 +1,25 @@
+## Part-of-Speech Tagging
+Trains a simple sequence-based part-of-speech tagger. The following sentence
+shows an example.
+
+   From|ADP the|DT AP|PROPN comes|VBZ this|DT story|NN :|:
+   
+
+### Requirements
+Universal Dependency data sets:  https://universaldependencies.org/#download.
+
+Download via command line: 
+
+    curl -# -o ud-treebanks-v2.0.tgz https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11234/1-1976/ud-treebanks-v2.0.tgz
+    tar xzf ud-treebanks-v2.0.tgz 
+
+### Supported setups
+The model should run with other configurations and hardware, but explicitly tested on the following.
+
+| Hardware |  Batch size  | Learning rate | Steps/Second | Accuracy  | TensorBoard |
+|:---:|:---:|:---:|:---:|:---:|:---:|
```

Comment:
Is this indented correctly? It looks a bit strange.

Reply:
|:---:| means center and shows the content better separated.

Reply:
Ahh I didn't know that markdown, nice :)